homeassistant/components/swiss_public_transport/const.py

```diff
@@ -0,0 +1,19 @@
+"""Constants for the swiss_public_transport integration."""
+
+DOMAIN = "swiss_public_transport"
+
+ATTR_DEPARTURE_TIME1 = "next_departure"
+ATTR_DEPARTURE_TIME2 = "next_on_departure"
+ATTR_DURATION = "duration"
+ATTR_PLATFORM = "platform"
+ATTR_REMAINING_TIME = "remaining_time"
+ATTR_START = "start"
+ATTR_TARGET = "destination"
+ATTR_TRAIN_NUMBER = "train_number"
+ATTR_TRANSFERS = "transfers"
+ATTR_DELAY = "delay"
```

Comment:
These are only used in `sensor.py` so should remain there and not be moved to here.